### PR TITLE
[eloot]v1.6.3 update for dropping boxes in shattered, excluded shimme…

### PR DESCRIPTION
**Pull request by Deysh**
…ring orbs from sell list, misc bug fixes

### CODE REVIEW

File: scripts/eloot.lic [hunk 1]
```diff
@@ -4160,7 +4170,7 @@ module ELoot
         next if $sell_ignore.include?(thing.id)
         next if ELoot.data.ready_list.find { |_k, v| v.id == thing.id }
         next if !ELoot.data.settings[:sell_exclude].empty? && thing.name =~ /#{ELoot.data.settings[:sell_exclude].join('|')}/
-        next if thing.name =~ /bound/
+        next if thing.name =~ /bound|shimmering \w+ orb/
```
Comment (mrhoribu) [inline scripts/eloot.lic:4173]: Wondering if these should be wrapped in appropriate anchors, so becomes:
```ruby
next if thing.name =~ /\bbound\b|^shimmering \w+ orb$/

**Review by mrhoribu — CHANGES_REQUESTED** [review]
Just a minor thing, but otherwise, good changes!

File: scripts/eloot.lic (hunk 1 above)
Comment (Deysh) [reply scripts/eloot.lic]: Agree, that makes sense.

**Review by mrhoribu — APPROVED** [review]
lgtm

**Pull request merged**